src/utils/translations/fa.ts

```diff
@@ -0,0 +1,96 @@
+const fa = {
+  // Home Tab
+  Home: "خانه",
+  "Period in": "تا پریود بعدی",
+  "no info": "بدون اطلاعات",
+  Period: "پریود",
+  today: "امروز",
+  Delay: "تاخیر",
+  Days: "(0)[روز];(1)[روز];(2-4)[روز];(5-20)[روز];(21)[روز];(22-24)[روز];(25-30)[روز];(31)[روز];(32-34)[روز];(35-40)[روز];",
```

Comment:
You dont need to write all these options in this language, instead you can use:

```ts
  Days: "(0-inf)[روز]",
```

Reply:
Done boss!